### PR TITLE
[Excel] (Custom Functions) Update the JSON metadata reference Options table

### PR DESCRIPTION
**Pull request by alison-mk**
This PR: 
- addresses the issue reported in #2306 
- removes an outdated link
- adjusts phrasing so code ticks can be used around property names, for clarity

### CODE REVIEW

**Comment by o365devx** [comment]
Docs Build status updates of commit _[b3569f0](https://github.com/OfficeDev/office-js-docs-pr/commits/b3569f0555a1f27299e98d49c814b52d629c6f02)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/excel/custom-functions-json.md](https://github.com/OfficeDev/office-js-docs-pr/blob/alison-mk-cf-metadata-options/docs/excel/custom-functions-json.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/excel/custom-functions-json?branch=pr-en-us-2367) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C12%5C8%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C202012080134454074-2367%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=fnXkTnzCndq4K470S7hfkwRsEoywnY5CEjJ89cyjzws%3D&st=2020-12-08T01%3A30%3A22Z&se=2021-01-08T01%3A35%3A22Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/57856ef7-e8cf-54ca-0d12-df5f42d1e8bb?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching in the <a href="https://review.docs.microsoft.com/en-us/search/?search=&amp;category=All&amp;scope=help-docs&amp;category=All&amp;branch=master">Docs contributor and Admin Guide</a></li><li>See the <a href="https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master">frequently asked questions</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

**Comment by o365devx** [comment]
Docs Build status updates of commit _[20d588f](https://github.com/OfficeDev/office-js-docs-pr/commits/20d588f3e1bb419c05f32893306196529dc6f7e0)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/excel/custom-functions-json.md](https://github.com/OfficeDev/office-js-docs-pr/blob/alison-mk-cf-metadata-options/docs/excel/custom-functions-json.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/excel/custom-functions-json?branch=pr-en-us-2367) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C12%5C8%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C202012080137426464-2367%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=SNzndc8XvgHvnmmYQiGsvOZvtuvndS3eoVoNyH%2Fh3a0%3D&st=2020-12-08T01%3A33%3A16Z&se=2021-01-08T01%3A38%3A16Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/57856ef7-e8cf-54ca-0d12-df5f42d1e8bb?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching in the <a href="https://review.docs.microsoft.com/en-us/search/?search=&amp;category=All&amp;scope=help-docs&amp;category=All&amp;branch=master">Docs contributor and Admin Guide</a></li><li>See the <a href="https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master">frequently asked questions</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

**Comment by o365devx** [comment]
Docs Build status updates of commit _[3dcd49e](https://github.com/OfficeDev/office-js-docs-pr/commits/3dcd49eff902faf8bf9d99730483b93afcd1a56d)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/excel/custom-functions-json.md](https://github.com/OfficeDev/office-js-docs-pr/blob/alison-mk-cf-metadata-options/docs/excel/custom-functions-json.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/office/dev/add-ins/excel/custom-functions-json?branch=pr-en-us-2367) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C12%5C8%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C202012080140027895-2367%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=ZY28jaomppKglvSjg2azKVmoMQ7pu4EZd%2BMArzaDmhU%3D&st=2020-12-08T01%3A35%3A45Z&se=2021-01-08T01%3A40%3A45Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/57856ef7-e8cf-54ca-0d12-df5f42d1e8bb?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching in the <a href="https://review.docs.microsoft.com/en-us/search/?search=&amp;category=All&amp;scope=help-docs&amp;category=All&amp;branch=master">Docs contributor and Admin Guide</a></li><li>See the <a href="https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master">frequently asked questions</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

File: docs/excel/custom-functions-json.md [hunk 1]
```diff
@@ -153,10 +153,10 @@ The `options` object enables you to customize some aspects of how and when Excel
...
+| `cancelable`      | boolean   | No<br/><br/>Default value is `false`.  | If `true`, Excel calls the `CancelableInvocation` handler whenever the user takes an action that has the effect of canceling the function; for example, manually triggering recalculation or editing a cell that is referenced by the function. Cancelable functions are typically only used for asynchronous functions that return a single result and need to handle the cancellation of a request for data. A function can't use both the `stream` and `cancelable` properties. |
+| `requiresAddress` | boolean   | No <br/><br/>Default value is `false`. | If `true`, your custom function can access the address of the cell that invoked your custom function. To get the address of the cell that invoked your custom function, call the `address` property of the invocation object. For example, if your invocation object is named `invocation`, call `invocation.address` in your custom function. When using `requiresAddress`, the `invocation` parameter must be the last parameter passed to the function. A function can't use both the `stream` and `requiresAddress` properties. |
```
Comment (davidchesnut) [inline docs/excel/custom-functions-json.md:157]:
```suggestion
| `requiresAddress` | boolean   | No <br/><br/>Default value is `false`. | If `true`, your custom function can access the address of the cell that invoked your custom function. The 'address' property of the [invocation](custom-functions-parameter-options.md#invocation-parameter) parameter contains the address of the cell that invoked your custom function. A function can't use both the `stream` and `requiresAddress` properties. |
```
I added a link to the docs for invocation parameter so we don't need to repeat info like (it's the last parameter) and so on... Need to double-check that this link builds correctly though.

**Review by davidchesnut — CHANGES_REQUESTED** [review]
These changes definitely help the readability! I made one suggestion for adding a link to help shorten the text...

**Review by davidchesnut — APPROVED** [review]
Looks great!

**Pull request merged**